app/Books.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Books extends Model
```

Comment:
Models devem ter o nome no singular.

```suggestion
class Book extends Model
```

Reply:
Feito